src/proxy/query.go

```diff
@@ -342,9 +345,16 @@ func queryStat(node sqlparser.Statement, err error) {
 	default:
 		command = "Unsupport"
 	}
+	queryTime := time.Since(timeStart)
 	if err != nil {
+		if queryTime > slowQueryTime {
+			monitor.SlowQueryTotalCounterInc(command, "Error")
```

Comment:
You need add test in the query_test.go to the logic which is core in slowlog, and the logic is not coveraged.
it is strange to that the travis-ci is passed

Reply:
I set long query time to 0 here and the success case is here:
https://github.com/radondb/radon/pull/94/files#diff-a59414309e8e470924b254be6b6f8c7eR111